Network/XmlRpc/Client.hs

```diff
@@ -164,7 +164,7 @@ post url headers content = do
 post_ :: URI -> URIAuth -> HeadersAList -> BSL.ByteString -> IO U.ByteString
 post_ uri auth headers content = withOpenSSL $ do
     let hostname = BS.pack (uriRegName auth)
-        port     = fromMaybe 443 (readMaybe $ tail $ uriPort auth)
+        port     = fromMaybe 443 (readMaybe $ tailSafe $ uriPort auth)
```

Comment:
Instead of implementing a helper function tailSafe you can just use `drop 1` which has the same behavior.